node/src/chain_spec/turing.rs

```diff
@@ -174,7 +179,25 @@ fn testnet_genesis(
 		valve: ValveConfig { start_with_valve_closed: false, closed_gates: pallet_gates_closed },
 		vesting: VestingConfig { vesting_schedule },
 		xcmp_handler: XcmpHandlerConfig { chain_data: xcmp_handler_data },
-		asset_registry: Default::default(),
+		asset_registry: AssetRegistryConfig {
+			assets: vec![(
+				0,
+				orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
+					&orml_asset_registry::AssetMetadata {
+						decimals: TOKEN_DECIMALS,
+						name: "Native".as_bytes().to_vec(),
+						symbol: TOKEN_SYMBOL.as_bytes().to_vec(),
+						existential_deposit: 100_000_000,
+						location: Some(V1(MultiLocation { parents: 0, interior: Here })),
+						additional: CustomMetadata {
+							fee_per_second: Some(416_000_000_000),
```

Comment:
These numbers were all taken from the current chain state on Turing.

Reply:
Looks good 👍